kubernetes/helm/ckeditor-docx-converter/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
+# This supports shell glob matching, relative path matching, and
```

Comment:
global?

Reply:
glob its fine, its a kind of patterns used commonly in file paths